src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,48 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        File fromFile = new File(fromFileName);
+        int supply = 0;
+        int buy = 0;
+        int result;
+        String lineSplitter = ",";
+        String supplyString = "supply";
+        try (BufferedReader lineReader = new BufferedReader(new FileReader(fromFile))) {
+            String line = lineReader.readLine();
+            while (line != null) {
+                if (supplyString.equals(line.split(lineSplitter)[0])) {
+                    supply += Integer.parseInt(line.split(lineSplitter)[1]);
```

Comment:
- use local variable for line.split(",")
- move indices and delimiter to constants

what about my comments above?